addons/mrp/static/src/scss/mrp_fields.scss

```
@@ -1,3 +1,7 @@
 .o_field_widget.o_embed_url_viewer iframe {
     aspect-ratio: 3/2;
 }
+
+.o_row .qty_producing_field {
```

when I test manually, it doesn't work because the CSS:

```py
.o_form_view .o_form_editable .o_row > div {
    width: 0 !important;
}
```
has higher priority. the order of priority is:

- does is it have !important?
- if both rules have equal priority: is it inline in a style tag?
- if both rules still have equal priority: is the CSS specificity (here is a calculator https://specificity.keegan.st/) higher? 
- if both rules still have equal priority: the last rules (in order of CSS sheets and order in the same css sheet) is taken into account

so to fix this, you could prefix for example the rule by ".o_form_view .o_form_editable", eg:

```suggestion
.o_form_view .o_form_editable .o_row .qty_producing_field {
```

---

thank you, done.